### ISSUE TITLE
lnd_test.go:69: Failed: (test multi-hop htlc remote chain claim)

### ISSUE THREAD

**Issue by JohnnyBackflip** [lightningnetwork/lnd#1410]
### Background

I get this Error when running make check:

>--- FAIL: TestLightningNetworkDaemon/test_multi-hop_htlc_remote_chain_claim (7.10s)
    	lnd_test.go:69: Failed: (test multi-hop htlc remote chain claim): exited with error: 
    		*errors.errorString commit transaction not spending fundingtx: (*btcutil.Tx)(0xc420e38ff0)({
    		 msgTx: (*wire.MsgTx)(0xc4208b4680)({
    		  Version: (int32) 2,
    		  TxIn: ([]*wire.TxIn) (len=1 cap=1) {
    		   (*wire.TxIn)(0xc4205db4a0)({
    		    PreviousOutPoint: (wire.OutPoint) de4ec37b15f2a54c4c785ae617850cebff6ab80efc2825f1b8496f42f0a1ba5b:1,
    		    SignatureScript: ([]uint8) {
    		    },
    		    Witness: (wire.TxWitness) (len=3 cap=3) {
    		     ([]uint8) (len=72 cap=72) {
    		      00000000  30 45 02 21 00 ae ec a5  e8 b1 5a b4 93 f0 d6 c8  |0E.!......Z.....|
    		      00000010  f3 dd c9 30 06 0f 4f ac  27 1e 21 74 94 85 74 0f  |...0..O.'.!t..t.|
    		      00000020  96 20 34 26 d9 02 20 0d  a6 dd 8b 60 6c e5 98 d0  |. 4&.. ....`l...|
    		      00000030  cf ea d7 22 7a 51 83 5a  58 d6 4a 8c 23 d4 cc 50  |..."zQ.ZX.J.#..P|
    		      00000040  c5 13 ca 82 9c 0b 9f 01                           |........|
    		     },
    		     ([]uint8) {
    		     },
    		     ([]uint8) (len=75 cap=75) {
    		      00000000  63 21 02 e3 b2 fe 2f 81  af 8f 82 a4 11 39 d4 98  |c!..../......9..|
    		      00000010  0b 56 38 4f aa 75 60 1c  aa 50 b5 97 4f d9 f3 c1  |.V8O.u`..P..O...|
    		      00000020  a5 62 d3 67 54 b2 75 21  02 4e 27 98 bc c8 04 66  |.b.gT.u!.N'....f|
    		      00000030  78 cf c2 16 06 78 f5 3e  87 86 59 98 b3 68 4c f0  |x....x.>..Y..hL.|
    		      00000040  a5 2e f7 e3 8f 88 55 9e  a5 68 ac                 |......U..h.|
    		     }
    		    },
    		    Sequence: (uint32) 4
    		   })
    		  },
    		  TxOut: ([]*wire.TxOut) (len=1 cap=1) {
    		   (*wire.TxOut)(0xc420dc3320)({
    		    Value: (int64) 882648,
    		    PkScript: ([]uint8) (len=22 cap=22) {
    		     00000000  00 14 cb 21 20 e0 47 41  b4 8d 5f 4a c1 f2 73 98  |...! .GA.._J..s.|
    		     00000010  44 db 39 ff 80 50                                 |D.9..P|
    		    }
    		   })
    		  },
    		  LockTime: (uint32) 0
    		 }),
    		 txHash: (*chainhash.Hash)(<nil>),
    		 txHashWitness: (*chainhash.Hash)(<nil>),
    		 txHasWitness: (*bool)(<nil>),
    		 txIndex: (int) -1
    		})
    		/home/johnnyb/go/src/github.com/lightningnetwork/lnd/lnd_test.go:8151 (0xb8b7f3)
    			testMultiHopHtlcRemoteChainClaim: t.Fatalf("commit transaction not spending fundingtx: %v",
    		/home/johnnyb/go/src/github.com/lightningnetwork/lnd/lnd_test.go:94 (0xb59a7b)
    			(*harnessTest).RunTestCase: testCase.test(net, h)
    		/home/johnnyb/go/src/github.com/lightningnetwork/lnd/lnd_test.go:10227 (0xba46e9)
    			TestLightningNetworkDaemon.func3: ht.RunTestCase(testCase, lndHarness)
    		/usr/lib/go-1.10/src/testing/testing.go:777 (0x4ef310)
    			tRunner: fn(t)
    		/usr/lib/go-1.10/src/runtime/asm_amd64.s:2361 (0x45c021)
    			goexit: BYTE	$0x90	// NOP
FAIL
exit status 1
FAIL	github.com/lightningnetwork/lnd	341.391s
Makefile:146: recipe for target 'itest' failed
make: *** [itest] Error 1

### Your environment

* commit 4344
* Ubuntu 18.04 LTS
* bitcoind v0.16.0.0-g4b4d7eb255

### Steps to reproduce

run "make check" on my system.

### Expected behaviour

No errors

**Comment by Roasbeef** [comment]
What commit are you running? Can you fill in the rest of the issue template?

**Comment by JohnnyBackflip** [comment]
Sry, im on commit 4344

**Comment by Roasbeef** [comment]
You'll need to provide a hash prefix longer than that.

**Comment by JohnnyBackflip** [comment]
For you? For lnd? For me? What 's a hash prefix in this context?

**Comment by Roasbeef** [comment]
What commit are you running? 4344 doesn't uniquely identify a commit.

**Comment by JohnnyBackflip** [comment]
>commit 7c2307984d38ed5a27f27798f30aacb619927454 (HEAD -> master, origin/master, origin/HEAD)
Merge: 2f6ff266 92711c4d
Author: Olaoluwa Osuntokun <laolu32@gmail.com>
Date:   Fri Jun 15 16:49:58 2018 -0700
Merge pull request #1380 from cfromknecht/nursery-rebroadcast
utxonursery: rebroadcast finalized kinder txns on startup

Sry, I hope thats better now

**Comment by Roasbeef** [comment]
Yes. Now we know which precise commit you're on. Can you fill in the rest of the template? Particularly: if you're able to reproduce this reliably and also some information concerning the machine that you're using.

**Comment by JohnnyBackflip** [comment]
Ok, so I'm on Ubuntu 18.04 LTS using bitcoind v0.16.0.0-g4b4d7eb255 .

What I did was following the installation guide here: https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md

I had the problem that I was missing permissions all the time. So I granted myself sudo permissions but this didn't work... Then I looked what folders were causing the errors and I granted myself permissions for them. After that "make install" seemed to work but "make check" fails.
